src/main/scala/com.snowplowanalytics.sauna/observers/AmazonS3Observer.scala

```diff
@@ -130,19 +134,19 @@ object AmazonS3Observer {
    */
   private[observers] def extractBucketAndFile(body: JsValue): Option[(String, String)] = {
     for {
-      s3     <- (body \ "Records" \\ "s3").headOption
+      s3 <- (body \ "Records" \\ "s3").headOption
       bucket <- (s3 \ "bucket" \ "name").asOpt[String]
-      file   <- (s3 \ "object" \ "key").asOpt[String]
+      file <- (s3 \ "object" \ "key").asOpt[String]
     } yield (URLDecoder.decode(bucket, "UTF-8"), file)
   }
 
   def props(s3: S3, sqs: SQS, queue: Queue): Props =
     Props(new AmazonS3Observer(s3, sqs, queue))
 
-  def props(parameters: AmazonS3ConfigParameters): Props = {
+  def props(parameters: AmazonS3ConfigParameters_1_0_0): Props = {
     // AWS configuration. Safe to throw exception on initialization
     val region = Region(parameters.awsRegion)
-    val credentials = new Credentials(parameters.awsAccessKeyId, parameters.awsAccessKeyId)
```

Comment:
Could you please rephrase and ammend Grzegorz' commit title to be consistent with other our commit. Something like `Fix S3 credentials provider`.